lib/spaceship/tunes/application.rb

```diff
@@ -139,6 +139,13 @@ def details
         Tunes::AppDetails.factory(attrs)
       end
 
+      # Lists all the reviews for a specifc store front
+      # @param store_front (StoreFront): The ISO 3166-1 Alpha-2 country code or "all" to get all the reviews
+      def reviews(platform="ios", store_front="US")
```

Comment:
* why isn't platform fetched from the application itself ?
* as for store_front I am not sure we want to have a default, or maybe "all" ?